### PR TITLE
Bump CNI to v0.5.1

### PR DESCRIPTION
**Pull request by luxas**
@ethernetdan @calebamiles @mikedanese @jbeda @kad 

Similar to @calebamiles' PR to Kubernetes.

### CODE REVIEW

**Comment by mikedanese** [comment]
Please link @calebamiles PR.

**Comment by mikedanese** [comment]
@freehan

**Comment by luxas** [comment]
Thanks @mikedanese! I guess you found it already, but here it is: https://github.com/kubernetes/kubernetes/pull/43546

File: debian/build.go [hunk 1]
```diff
@@ -321,17 +321,17 @@ func main() {
 			Distros: serverDistros,
 			Versions: []version{
 				{
-					Version:  "0.3.0.1-07a8a2",
+					Version:  "0.3.0.1-0799f5",
```
Comment (mikedanese) [inline debian/build.go]: Shouldn't these be change to 0.5.1-0799f5

Comment (luxas) [inline debian/build.go]: ooh, missed that number, fixing in a minute

**Pull request merged**